magento/pwa-studio
# [feature]: Debugging Reporter

## Description

This feature/hackathon project should make it easier for folks to provide debugging info about their Magento/PWA Studio system.

I was not able to access extension or target usage info. I wasn't sure what the optimal way to do this was. Perhaps there is some tooling we have already written that exposes it.

## Related Issue





Closes [PWA-1091](https://jira.corp.magento.com/browse/PWA-1091).

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Scaffold a new app using this branch
    `DEBUG_PROJECT_CREATION=1 ../pwa-studio/packages/create-pwa/bin/create-pwa`
2. In the scaffold, `yarn install` and `yarn build:report`
3. See output!

## Screenshots / Screen Captures (if appropriate)

[![Image from Gyazo](https://i.gyazo.com/960b390007953778d67d298bd27b4886.png)](https://gyazo.com/960b390007953778d67d298bd27b4886)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
